### ISSUE TITLE
[Issue]: Unit for dummy chapter interval setting not documented

### ISSUE THREAD

**Issue by DrMcCoy** [jellyfin/jellyfin-web#5555]
### Please describe your bug

The help text for the "Interval" setting for the dummy chapters in Libraries -> Metadata does not mention any units. Is it in seconds? Milliseconds? Minutes? Fortnights?

### Reproduction Steps

1. Open Jellyfin
2. Dashboard 
3. Libraries
4. Metadata
5. Look at the help text for "Interval"

### Jellyfin Version

10.9.0

### if other:

10.9.2

### Environment

```markdown
-
```

### Jellyfin logs

```shell
-
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by webysther** [comment]
Just a question, today is general, but looks like dummy chapter will be better to be in the settings of the library because based on the lenght of content 120s will be better and for bigger video files 5 minutes make sense?

**Comment by freeotoole** [comment]
Just looking for a good first issue and noticed this is still open and the issue still exists.

<img width="836" alt="Image" src="https://github.com/user-attachments/assets/0fc0a0d0-086d-4412-9c7a-b64289e77a9f" />

I spotted this open PR from last October that hasn't been merged - https://github.com/jellyfin/jellyfin-web/pull/6158

Just wondering if this issue is something I can tackle? And if so, what's the best process? Can I just create a new PR and link the issues if approved?

**Comment by thornbill** [comment]
@freeotoole yeah this would be a great first issue! Really all that needs to be changed is to add the unit to the text in `LabelDummyChapterDuration` in the en-us file **only** ("The interval between dummy chapters **in seconds**..."). Other languages are managed via our weblate instance (translate.jellyfin.org). The other PR can be considered abandoned since the feedback was never addressed.
